.nvmrc

```diff
@@ -0,0 +1 @@
+v20.10.0
```

Comment:
no, I think it was easier for me to pin to that version because it's the one I was using in `ui-monorepo` (so `npm link ` would work). Nvm would create 2 different "virtual" spaces for each node version, and I was unable to link one from another. I can update it

Reply:
Pinned to `20` here [e5aa862](https://github.com/hemilabs/umami-analytics-next/pull/1/commits/e5aa862bceedcbcba8696b1109473469585b78cb)